3scale/porta
# THREESCALE-10167: Formula for cgroups v2

**What this PR does / why we need it**:

Support Cgroups v2 for auto-detection of unicorn workers

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-10167

**Verification steps** 

Run on OCP 4.14 (that uses cgroups v2 by default) and make sure that the unicorn workers number is as expected (according to the CPU requests/limits)

**Special notes for your reviewer**:

References: 
https://linuxera.org/cpu-memory-management-kubernetes-cgroupsv2/#cpu-bandwidth-configuration-on-the-node
https://github.com/kubernetes/kubernetes/blob/release-1.27/pkg/kubelet/cm/helpers_linux.go#L45
https://github.com/kubernetes/kubernetes/blob/release-1.27/pkg/kubelet/cm/cgroup_manager_linux.go#L570-L574
